cmd/server/assets/realmkeys.html

```diff
@@ -0,0 +1,262 @@
+            <div class="col-sm-9">
+              <input type="text" id="iss" class="form-control" value="{{.certIssuer}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Audience (aud):</label>
+            <div class="col-sm-9">
+              <input type="text" id="aud" class="form-control" value="{{.certAudience}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Certificate duration:</label>
+            <div class="col-sm-9">
+              <input type="text" id="certDuration" class="form-control" value="{{.certDuration}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Key ID (kid):</label>
+            <div class="col-sm-9">
+              <input type="text" id="certKeyID" class="form-control" value="{{.certKeyID}}" readonly/>
+            </div>
+          </div>
+          <div class="form-group row">
+            <label for="name" class="col-sm-3">Public key:</label>
+            <div class="col-sm-9">
+            {{if .certPublicKey}}
+            <div class="alert alert-secondary" role="alert">
+            <pre>{{.certPublicKey}}</pre>
+            </div>
+            {{else}}
+            <div class="alert alert-danger" role="alert">{{.certPublicKeyError}}</div>
+            {{end}}
+            </div>
+          </div>
+        </div>
+      </div>
+    {{end}}
+      
+    <div class="card mb-3">
+        <div class="card-header">
+          Realm signing key configuration
+        </div>
+        <div class="card-body">
+        {{if .supportsPerRealmSigning}}            
+          {{if not $realm.UseRealmCertificateKey}}
+            {{if $realm.CanUpgradeToRealmSigningKeys}}
+            <form method="POST" action="/realm/keys/upgrade">
+                {{ .csrfField }}
+                <div class="form-group row">
+                  <div class="col-sm-12">
+                    <div class="alert alert-warning" role="alert">
+                        <strong>Unerversable action:</strong> Before upgrading to realm specific signing keys, make sure that you have
+                        configured the signing key information on your exposure notifications key server.
+                        Once you upgrade to realm spcific signing keys, this realm cannot be changed
+                        back to using the system signer. This change will be effective immediatly.
+                    </div>
+                    <button type="submit" class="btn btn-primary btn-block">Upgrade to realm specific signing keys</button>
+                  </div>
+                </div>
+            </form>
+            {{else}}
+            <div class="alert alert-info" role="alert">
+                After setting your realm's issuer and audience settings, you can enable
```

Comment:
I recommend dropping the "alert" since it's busy. Maybe:

> You can enable realm-specific verification certificates after configuring your realm's issuer and audience settings above.